src/randomizer.py

```diff
@@ -493,6 +493,29 @@ def rec(block: Block) -> None:
     return cands
 
 
+def get_noncolliding_fn_name(ast: ca.FileAST, fn_name: str) -> str:
+
+    used_names: Set[str] = set()
+
+    for item in ast.ext:
+        if (
+            isinstance(item, ca.Decl)
+            and isinstance(item.type, ca.FuncDecl)
```

Comment:
```suggestion
```
to generalize this function to not just cover functions

Reply:
done